### DISCUSSION TITLE
I can't see Galaxy Store app in AM - is this normal? Any way to fix? searching for search activities and registered search providers (like settings, and Galaxy Store)

### DISCUSSION THREAD

**Discussion by iamjonny** [sdex/ActivityManager#67] [Q&A]
Thank you so much for this amazing app!

I've just started module-testing on newly rooted S20, and one thing I'd like to know if possible is understanding the "search engines" that are registered when searching from phone search.

In order of appearance, I have: settings, widgets, contacts, spotify, and **galaxy store**
But not AMM , Obtainium, Chrome <super>💡</super> , Play store!!, F-Droid...

💡The default browser may be google, hence no chrome, if the searchable apps are related/similar to default apps... 

I thought AM may give clues as to how galaxy store does this, in case possible to register others, like `SearchEngines` in browsers like [Edge](edge://settings/searchEngines) , but as usual it's not that simple on droid!

Any insights welcome!

**Comment by sdex** [comment]
I don't know why you can't see it, it exists on my device. 

![photo1704028221(1)](https://github.com/sdex/ActivityManager/assets/3196011/0ba3f0e4-0eae-44b4-8fbc-4089f7739638)

As I understand, you're looking for search interface that apps can implement to provide the results:
https://developer.android.com/develop/ui/views/search/searchable-config

It seems like in the galaxy store it's `SettingsIndexablesSearchProvider`

![photo1704028221](https://github.com/sdex/ActivityManager/assets/3196011/61e3d6a9-be88-4aab-b826-0524b88d6721)

**Comment by iamjonny** [comment]
Thanks so much for this, I thought we'd find unexpected behaviour relating to **app status** - on samsung android 13 with onehome 5 you can hide apps but still locate them via finder (search). I've just retried, but even with Galaxy Store "visible" (not a hidden app), it's not seen in my AM (v5.4.5)

One thing to note, if I recall correctly there was an update to the store app, _via_ the store app - play store didn't report it as an app with available update... I could be mis-recalling the apps and stores, doubt any of that helps.

~~If I start some logcat record and then perform some steps could that help?~~ Solution much simpler - store is a system app, which I wasn't showing!

Thanks for assist !

**Comment by iamjonny** [comment]
> It seems like in the galaxy store it's SettingsIndexablesSearchProvider

How did you get to this (conclusion, and the manifest) ?

After finding the store, and searching for the **search** action (SettingsIndexablesSearchProvider) I have 3, including SearchResultActivity and SearchSettingsListActivity, but not the Provider mentioned... 

Thanks again!

**Reply by drogga** [reply]
To me it looks like you are trying to find a provider or any of the other components in A.M., which lists **only activities**, not providers or anything else, but if you view the manifest of the Galaxy Store (from A.M. of other apps, those that are able to open and show that file) you should be able to find the provider you are looking for (those are not launchable).

_No need to respond/reply to this, you can react to it if you want (GH thing)_ ;)
